NextcloudTalk/FileMessageTableViewCell.h

```diff
@@ -26,9 +26,15 @@
 #import "NCMessageFileParameter.h"
 #import "NCChatMessage.h"
 
-static CGFloat kFileMessageCellMinimumHeight        = 50.0;
-static CGFloat kFileMessageCellFilePreviewHeight    = 120.0;
-static CGFloat maxPreviewImageWidth                 = 230.0;
+static CGFloat kFileMessageCellMinimumHeight                = 50.0;
+static CGFloat kFileMessageCellFilePreviewHeight            = 120.0;
+static CGFloat kFileMessageCellFileMaxPreviewWidth          = 230.0;
+static CGFloat kFileMessageCellMediaFilePreviewHeight       = 230.0;
```

Comment:
Should we go for `...MaxPreviewHeight` here? (I would also be fine to `MediaFile -> Media` to not make it any longer)

Reply:
Okay, let's see if I can make these names clearer:
1. `kFileMessageCellMinimumHeight` is declared but never used, should I still keep it?
<img width="278" alt="Screenshot 2022-10-29 at 18 32 23" src="https://user-images.githubusercontent.com/48513706/198842742-fb25e79c-7201-4649-85ab-2d363da16831.png">

2. `kFileMessageCellFilePreviewHeight` is used to determine the initial square size (`kFileMessageCellFilePreviewHeight`x`kFileMessageCellFilePreviewHeight`) of the file cell image preview. It is also the maximum height of the (not media) file cell, so `kFileMessageCellFileMaxPreviewHeight` makes sense.
3. `kFileMessageCellFileMaxPreviewWidth` is only used to limit the width of the preview, so the name already makes sense as it is
4. I've named `kFileMessageCellMediaFilePreviewHeight` like this in order to differentiate a `File` from a `MediaFile`. If you prefer I can drop the `File`.
5. same for `kFileMessageCellMediaFileMaxPreviewWidth`